### PR TITLE
Added option to define text separators to use

### PR DESCRIPTION
**Pull request by ufobm**

### CODE REVIEW

**Comment by eyoeldefare** [comment]
Thanks for the contribution, I will review this very soon.

**Comment by eyoeldefare** [comment]
Hey there, tests haves failed for entering tags with comma, and entering tags with space-bar out of the box. What exactly is this trying to achieve and please test all functionalities locally first before merge request. 
Thanks for contributing

**Comment by ufobm** [comment]
Are you talking about manual tests? I don't think automatic tests are implemented. Of course, comma and space-bar remains the defaults text separators! I added an option to define different text separators, like so:

textSeparators: [".", "!"],

I removed this line from the example, so now it should work as expected out of the box.

**Comment by eyoeldefare** [comment]
Yes, I am manually testing the functionalities all around. 
To cut down on time, before I examine the changes, I simply check if the app works out of the box with new changes. So i appreciate a little description on what new pull requests do.
Regardless, I will test this again over this weekend. 
Thanks

File: lib/src/main.dart [hunk 1]
```diff
@@ -231,18 +240,22 @@ class _TextFieldTagsState extends State<TextFieldTags> {
...
                 : splitedTagsList.length - 1;
             final String lastLastTag =
                 splitedTagsList[indexer].trim().toLowerCase();
             if (lastLastTag.length > 0) {
-              _textEditingController!.clear();
+              _textEditingController.clear();
               if (!_tagsStringContents!.contains(lastLastTag)) {
-                if (widget.validator == null ||
-                    widget.validator!(lastLastTag) == null) {
+                final validatorResult = widget.validator!(lastLastTag);
```
Comment (eyoeldefare) [inline lib/src/main.dart]: Reverse this because if user doesn't implement `validator`, it gives of `null` where you can't `null!`.

**Review by eyoeldefare — COMMENTED** [review]
Overall, looks good to me.

File: lib/src/main.dart (hunk 1 above)
Comment (ufobm) [reply lib/src/main.dart]: Done! Thank you!

**Comment by ufobm** [comment]
Thank you! Will this merge be published on pub dev?

**Comment by eyoeldefare** [comment]
hey, yeah I will publish it this weekend. 
Eyoel

**Pull request merged**